docs/topics/i18n/translation.txt

```diff
@@ -730,6 +730,25 @@ will result in the entry ``"First sentence. Second paragraph."`` in the PO file,
 compared to ``"\n  First sentence.\n  Second sentence.\n"``, if the ``trimmed``
 option had not been specified.
 
+Also ``{% blocktrans %}`` has option `force_escape` to force HTML escaping for
+translated content. It behaves the same as if you applied :tfilter:`force_escape`
+filter, e.g.:
+
+.. code:: html+django
```

Comment:
we're already in "highlightlang:: html+django" so we can simply use :: instead of ".. code:: html+django"